via/static/scripts/video_player/components/Transcript.tsx

```diff
@@ -347,9 +348,12 @@ export default function Transcript({
     return filterTranscript(transcript.segments, filter);
   }, [filter, transcript]);
 
+  useOverflowAnchor(scrollRef, element => element.querySelectorAll('li'));
```

Comment:
Don't forget to memo the callback, otherwise the effects internally will re-run on every render.